### ISSUE TITLE
Thermometer Icon

### ISSUE THREAD

**Issue by Mercerenies** [EvanSkiStudios/Raccoon-Mischief#12]
You mentioned it in passing at one point, having a thermometer on-screen that shows the temperature of the player. A Turtle's Troll has temperature damage as of now, so that feature would go nicely with this one.

For your reference, my plugin damages the player if
* They are in a biome with temperature > 0.8 and have armor on, or
* They are in a biome with temperature < 0.2 and have no armor on

It doesn't have any effect in the nether or the end, because holy god the nether is hard enough already.

**Comment by EvanSkiStudios** [comment]
I could implement this, but to get it to sync with the plugin I need more info about how exactly it decides to damage the player
the only way on my side I'd be able to edit it is with scoreboard commands

is it like an instant player is not wearing armor in cold take damage?

**Comment by Mercerenies** [comment]
Checks every 4 seconds. If it's a hot penalty, it just lights you on fire. If it's a cold penalty, it makes the powder snow visuals appear on screen and deals instant damage to you (apparently, the powder snow visuals don't deal damage like the fire does)

**Comment by EvanSkiStudios** [comment]
Is this in the latest commit?

**Comment by Mercerenies** [comment]
Yes, it's all in the `TemperatureManager` class.

**Comment by EvanSkiStudios** [comment]
alright, i''ll take a look at it when i get a chance and see what i can do

**Comment by EvanSkiStudios** [comment]
So it seems the plugin just has a set of cold biomes, and if the player is in them without armor they take damage every 4 secs
so in the means of a gui, you basically already got it with the freeze gui

What I could do, is have it just be a temp gadget for each biome, rather then a real, 
hey your taking damage because its cold, unless the plugin has changes to how it handles this

**What changes?:**
Each player has their own temp value
it decreases in cold biomes and increases in hot biomes
when that value less than or equal to  0 they take damage every 4 secs
if it is more or equal to 100 they take heat damage

with that style the temp gui would change dependent on the players temp value defined by the plugin (which would give it more use)

(If you do this, an easy for evan thing is to make the player value also be a scoreboard score for the player, which the datapack can look up and use)

so TL;DR
implement my idea change for the temp system

or I make a thing on the screen that tells you when you are in a hot or cold biome

**Comment by Mercerenies** [comment]
So Minecraft actually does keep a block-local temperature, which is all I query. You can be in a biome and travel straight up the Y axis, and the temperature will decrease, which is why it can snow at the top of mountains but rain at the base of them, despite them being the same biome.

I can expose the Minecraft temperature value of the player as a scoreboard value if that would make it easier for you. It's literally `block.getTemperature()` on my end.

**Comment by EvanSkiStudios** [comment]
That would work, as I have no way on my end to check for that other then the scoreboard
